Desktop/components/JASP/Widgets/AnalysisFormExpander.qml

```diff
@@ -77,6 +79,15 @@ DropArea
 					anchors.left:	undefined
 					anchors.right:	undefined
 				}
+
+				PropertyChanges
+				{
+					restoreEntryValues: false
+					draggableItem
+					{
+						focus:			true
+					}
+				}
```

Comment:
I don't think you need to specify draggableItem here since the PropertyChanges is for a state of the draggableItem.
```
				PropertyChanges
				{
					restoreEntryValues: false
				    focus:			true
				}
```

Reply:
It seems necessary otherwise the error:
"QML PropertyChanges: Cannot assign to non-existent property "focus""
Is generated.
It doesn't hurt for the sake of clarity either.